### ISSUE TITLE
softmax_lastdim_kernel_impl" not implemented for 'Half'

### ISSUE THREAD

**Issue by YagaoDirac** [pytorch/pytorch#142368]
### 🚀 The feature, motivation and pitch

torch.float16, cpu, I called some tensor.softmax(dim=1), I got:
"softmax_lastdim_kernel_impl" not implemented for 'Half'

I know how to write this for myself. I mean, do you forget this detail, or bc of the precision of half and softmax has exp inside it, so it's not very recommended to call softmax with half?

If it's a feature, simply close this issue.🎉🎉

**Comment by malfet** [comment]
@YagaoDirac what's your python version?

**Comment by YagaoDirac** [comment]
> @YagaoDirac what's your PyTorch version? Half support was added in #103315 which I guess is part of 2.2+
Oh, that's probably the answer. I simply had fun offline for quite a long time and I forgot to update it and still don't plan to update it. Thanks. If I encounter any other problem I'll update pytorch and check out again before open any issue here again.
